Source/Orts.Simulation/Simulation/Timetables/TTTrain.cs

```diff
@@ -209,6 +209,7 @@ public struct TriggerActivation
         }
 
         public List<TriggerActivation> activatedTrainTriggers = new List<TriggerActivation>();
+        public string Briefing { get; set; } = "";
```

Comment:
```suggestion
        public string Briefing { get; } = "";
```
I can't imagine a scenario in which we would need to modify the briefing.

Reply:
Worked fine on my PC but broke the build with:

"C:\projects\openrails\Source\Orts.Simulation\Orts.Simulation.csproj" (default target) (8:10) ->
(CoreCompile target) -> 
  Simulation\Timetables\ProcessTimetable.cs(1601,21): error CS0200: Property or indexer 'TTTrain.Briefing' cannot be assigned to -- it is read only [C:\projects\openrails\Source\Orts.Simulation\Orts.Simulation.csproj]

I guess that shows the value of the build process.